aura/chain.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "aura",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://aura.network/",
+  "pretty_name": "Aura Network",
+  "chain_id": "xstaxy-1",
+  "bech32_prefix": "aura",
```

Comment:
please register this bech32 prefix here:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md